sources/packages/api/src/route-controllers/institution-locations/institution-locations.controller.ts

```diff
@@ -66,7 +66,6 @@ export class InstitutionLocationsController extends BaseController {
         "Not able to find a institution associated with the current user name.",
       );
     }
-
     // If the data is valid the location is saved to SIMS DB.
```

Comment:
Please consider do not include this file in the PR if there is no modifications related to this PR.